apps/frontend/src/components/ui/carousel.tsx

```
@@ -0,0 +1,241 @@
+        orientation:
+          orientation || (opts?.axis === 'y' ? 'vertical' : 'horizontal'),
+        scrollPrev,
+        scrollNext,
+        canScrollPrev,
+        canScrollNext,
+      }}
+    >
+      <div
+        onKeyDownCapture={handleKeyDown}
+        className={cn('relative', className)}
+        role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)